### ISSUE TITLE
Linux: Game crashes during opening cutscene

### ISSUE THREAD

**Issue by FloFluoro** [open-goal/jak-project#1793]
### Acknowledgements

- [X] I have [searched](https://github.com/open-goal/jak-project/issues?q=is%3Aissue+is%3Aopen+label%3Abug+sort%3Aupdated-desc) for duplicate or closed bug reports
- [X] I understand that I am supposed to provide my own legitimately obtained copy of the game

### Describe the Bug

OpenGOAL crashes during the opening cutscene where Jak and Daxter arrive at Misty Island, directly after the "Ol' green stuff told us not to come here" line.

The crash happens 100% of the time. Tested on Steam Deck running SteamOS 3.3, with OpenGOAL running as a non-Steam game in game mode. The native Linux version of OpenGOAL was used.

### How To Reproduce

1. On Steam Deck, in desktop mode, install the Linux version of OpenGOAL v0.1.25 using an NTSC 1.0 copy of the game
2. Add the executable as a non-Steam game
3. In game mode, launch and start a new game
4. The game will crash during the misty island cutscene

### Does this problem occur on original hardware or PCSX2?

Yes, it's unique to OpenGOAL

### Expected Behavior

The game should not crash

### Operating System

SteamOS 3.3

### OpenGOAL Version

v0.1.25

### Game Version

NTSC 1.0 (black label)

### Are you using a high refresh-rate monitor (>60Hz)

No

**Comment by xTVaser** [comment]
Likely a duplicate of https://github.com/open-goal/jak-project/issues/1792

Can you go into the `pc-settings.gc` file and change `(lod-force-tie 2)` to `(lod-force-tie 0)` and see if that prevents the crash?

**Comment by FloFluoro** [comment]
> Likely a duplicate of #1792
> 
> Can you go into the `pc-settings.gc` file and change `(lod-force-tie 2)` to `(lod-force-tie 0)` and see if that prevents the crash?

That setting was already at 0. Setting it to 2 only reproduced the crash seen in #1792 .

**Comment by ManDude** [comment]
Can you post your `pc-settings.gc` then?

**Comment by FloFluoro** [comment]
> Can you post your `pc-settings.gc` then?

Sure, [here's a link.](https://www.mediafire.com/file/bo5tgs8vcpr1u3i/pc-settings.gc/file)

**Comment by water111** [comment]
Thanks for the very detailed bug report - having information like this helps a ton.

I agree that it is unrelated to the TIE LOD issue.

I replicated this crash with your settings, and with the default settings. So I think it isn't related to settings either.

I got it 3 times in a row, so I think it's a deterministic problem.

The crash happens because we hit the `dummy-19 bad` print statement in `defmethod evaluate-joint-control` in `process-drawable.gc`. I'm not really sure how this could have broken, but I'll start bisecting to see if I can find the change.
